.github/workflows/ci.yml

```diff
@@ -90,19 +90,33 @@ jobs:
               cargo build
               ;;
           esac
+      - name: build verus release
+        if: matrix.features == ''
+        working-directory: ./source
+        run: |
+          . ../tools/activate
+          vargo clean
+          vargo build --release
+          ./target-verus/release/verus --version --output-json > ./target-verus/release/version.json
+      - name: upload verus release artifact
+        uses: actions/upload-artifact@v4
+        if: matrix.features == ''
+        with:
+            name: verus-arm64-macos
+            path: source/target-verus/release
```

Comment:
Might be good to add a quick test before this to confirm that it _is_ actually on arm64. We don't want to accidentally get x86-64 binaries being called arm64 (Rosetta will prevent it from being _immediately_ obvious, but it will end up with folks getting slower Verus runs, which is not a good experience)

Reply:
Do you have a command in mind to do this?

Reply:
`uname -m` is portable, and I believe should work (am not an an arm machine right now to test though)